backend/campaigns/tests/test_admin_permissions.py

```
@@ -0,0 +1,69 @@
+        self.marketer.groups.add(self.group)
+        self.client = Client()
+        self.client.force_login(self.marketer)
+
+    def test_group_has_campaign_permissions_only(self):
+        codenames = set(self.group.permissions.values_list('codename', flat=True))
+        self.assertIn('add_marketingcampaign', codenames)
+        self.assertIn('change_campaignlink', codenames)
+        self.assertIn('view_campaignredirecthit', codenames)
+        self.assertIn('view_useracquisitionattribution', codenames)
+        self.assertNotIn('delete_marketingcampaign', codenames)
+        self.assertFalse(any('user' == c.split('_', 1)[1] for c in codenames))
```

Ruff (0.16.0)

[warning] Yoda condition detected

Rewrite as `c.split('_', 1)[1] == 'user'`

(SIM300)